pkg/quota/v1/install/registry.go

```diff
@@ -41,6 +41,7 @@ var ignoredResources = map[schema.GroupResource]struct{}{
 	{Group: "", Resource: "bindings"}:                                      {},
 	{Group: "", Resource: "componentstatuses"}:                             {},
 	{Group: "authentication.k8s.io", Resource: "tokenreviews"}:             {},
+	{Group: "authentication.k8s.io", Resource: "selfsubjectaccessreviews"}: {},
```

Comment:
this name looks wrong to me. You meant selfsubjectattributereviews?

Reply:
My bad, fixed.